tasks/config.yml

```diff
@@ -342,6 +342,24 @@
   - restart freeswitch
   - reload nginx
 
+- name: kurento only listen on specified interfaces
+  replace:
+    path: /etc/kurento/modules/kurento/WebRtcEndpoint.conf.ini
+    regexp: '^;?networkInterfaces=.*$'
+    replace: 'networkInterfaces={{ bbb_kurento_interfaces | join(",") }}'
+  when: (bbb_kurento_interfaces | default([]) | length) > 0
+  notify: restart kurento
+  tags: debug
```

Comment:
I think the `tags: debug` are still a leftover from bugging? :D
Removing them would be nice.